src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -181,7 +181,16 @@ export class LogsJsonScene extends SceneObjectBase<LogsJsonSceneState> {
       } else {
         this.setVizFlags(detectedFieldFrame);
       }
+    } else if (serviceScene.state?.$detectedFieldsData?.state.data?.state === undefined) {
```

Comment:
I guess if we wanted an abundance of caution we should throw a console warning if the service scene doesn't have a $detectedFieldsData query runner